src/setup-bot.ts

```diff
@@ -73,6 +78,29 @@ export async function setupBot (): Promise<void> {
     }),
   )
 
+  if (process.env['CHATOPERA_CLIENTID'] && process.env['CHATOPERA_SECRET']) {
+    const chatoperaFiltersGroup = process.env['CHATOPERA_FILTERS_GROUP']?.split(',')
+
+    haWechaty.use(
+      WechatyChatopera({
+        mention: false,
+        room: async (room: Room): Promise<boolean> => {
```

Comment:
Our Wechaty plugin `[RoomMatcher](https://github.com/wechaty/wechaty-plugin-contrib/blob/16b69428cc486b49f5e0f897c0347ba30605df52/src/matchers/room-matcher.spec.ts#L122-L125)` supports the following syntax: (with a RegExp list to match room topics)

```ts
room: [
  new RegExp(topicFilter1),
  new RegExp(topicFileter2),
],
```

The above code can be more readable and easy to be understand by replacing the blow code (a `matchFunction`)

Reply:
I hope this information helps.